substrate/frame/tips/src/lib.rs

```diff
@@ -171,6 +171,9 @@ pub mod pallet {
 
 		/// Weight information for extrinsics in this pallet.
 		type WeightInfo: WeightInfo;
+
+		/// Handler for the unbalanced decrease when slashing for a bounty.
```

Comment:
for a tip

Reply:
@muharem Thanks, I made an update here: https://github.com/openguild-labs/polkadot-sdk/pull/3/commits/4031443ff7328173972b16f96f926737b646277e